### PR TITLE
docs: improve uploaded_files.rst into develop branch.

### LINKED ISSUE #7008: Undefined variable $uploaded_fileinfo on https://codeigniter4.github.io/userguide/libraries/uploaded_files.html#the-success-page
### PHP Version

8.1

### CodeIgniter4 Version

4.2

### CodeIgniter4 Installation Method

Composer (using `codeigniter4/appstarter`)

### Which operating systems have you tested for this bug?

Windows

### Which server did you use?

apache

### Database

MariaDB

### What happened?

I get error with file uploading

ErrorException
Undefined variable $uploaded_fileinfo
APPPATH\Views\common\upload_success.php at line 11

### Steps to Reproduce

I have created script by instructions in documentation -- https://codeigniter4.github.io/userguide/libraries/uploaded_files.html#creating-the-upload-form

I understand that there is an mistake in documentation near:

`$data = ['uploaded_flleinfo' => new File($filepath)];`
It need to be:
`$data = ['uploaded_fileinfo' => new File($filepath)];`

I mean there is char **l** instead of **i** in substring 'file'

### Expected Output

Return file info after uploading

### Anything else?

I have found mistake n documentation and described it

### PR DESCRIPTION
**Pull request by ping-yee**
**Description**
Fixes #7008 
Following the PR #6983 to fix this document typo.

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: user_guide_src/source/libraries/uploaded_files/002.php [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
 
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
```
Comment (kenjis) [inline user_guide_src/source/libraries/uploaded_files/002.php:11]: Please remove.

File: user_guide_src/source/libraries/uploaded_files.rst [hunk 2]
```diff
@@ -126,7 +126,7 @@ request, use ``getFiles()``. This will return an array of files represented by i
 Of course, there are multiple ways to name the file input, and anything but the simplest can create strange results.
 The array returns in a manner that you would expect. With the simplest usage, a single file might be submitted like::
 
-    <input type="file" name="avatar" />
+    <input type="file" name="avatar">
```
Comment (kenjis) [inline user_guide_src/source/libraries/uploaded_files.rst:129]: `4.3` improves HTML5 compatibility.
https://github.com/codeigniter4/CodeIgniter4/blob/4.3/user_guide_src/source/changelogs/v4.3.0.rst#html5-compatibility
But 4.2 is not.

Do you really need to these tag changes?

File: user_guide_src/source/libraries/uploaded_files/002.php (hunk 1 above)
Comment (ping-yee) [reply user_guide_src/source/libraries/uploaded_files/002.php]: Done.

File: user_guide_src/source/libraries/uploaded_files.rst (hunk 2 above)
Comment (ping-yee) [reply user_guide_src/source/libraries/uploaded_files.rst]: Ok, I got it and I have been reverted it.

**Review by ddevsr — APPROVED** [review]

**Review by kenjis — APPROVED** [review]

File: user_guide_src/source/libraries/uploaded_files.rst (hunk 2 above)
Comment (kenjis) [reply user_guide_src/source/libraries/uploaded_files.rst]: Thank you.

**Pull request merged**